modules/Bio/EnsEMBL/Analysis/Hive/Config/StarScallopRnaseq.pm

```diff
@@ -0,0 +1,827 @@
+
+    # Keys for custom loading, only set/modify if that's what you're doing
+    'protein_blast_db' => '' || catfile( $self->o('base_blast_db_path'), 'uniprot', $self->o('uniprot_version'), 'PE12_vertebrata' ),    # Blast database for comparing the final models to.
+    'protein_blast_index' => '' || catdir( $self->o('base_blast_db_path'), 'uniprot', $self->o('uniprot_version'), 'PE12_vertebrata_index' ),    # Indicate Index for the blast database.
+    'protein_entry_loc' => catfile( $self->o('base_blast_db_path'), 'uniprot', $self->o('uniprot_version'), 'entry_loc' ),                       # Used by genscan blasts and optimise daf/paf. Don't change unless you know what you're doing
+
+########################
+# Pipe and ref db info
+########################
+
+    'pipe_db_name' => $self->o('dbowner') . '_' . $self->o('production_name') . '_pipe_' . $self->o('release_number'),
+    'dna_db_name'  => $self->o('dbowner') . '_' . $self->o('production_name') . '_core_' . $self->o('release_number'),
+
+    'reference_db_name'   => $self->o('dna_db_name'),
+    'reference_db_server' => $self->o('dna_db_server'),
+    'reference_db_port'   => $self->o('dna_db_port'),
+
+    'star_rnaseq_for_layer_db_server' => $self->o('databases_server'),
+    'star_rnaseq_for_layer_db_port'   => $self->o('databases_port'),
+
+    'scallop_initial_db_server' => $self->o('databases_server'),
+    'scallop_initial_db_port'   => $self->o('databases_port'),
+
+    'scallop_blast_db_server' => $self->o('databases_server'),
+    'scallop_blast_db_port'   => $self->o('databases_port'),
+
+    # This is used for the ensembl_production and the ncbi_taxonomy databases
+    'ensembl_release' => $ENV{ENSEMBL_RELEASE},    # this is the current release version on staging to be able to get the correct database
+
+    databases_to_delete => ['reference_db'],
+
+########################
+# BLAST db paths
+########################
+    'base_blast_db_path' => $ENV{BLASTDB_DIR},
+
+######################################################
+#
+# Mostly constant settings
+#
+######################################################
+
+    genome_dumps => catdir( $self->o('output_path'), 'genome_dumps' ),
+    # This one is used in replacement of the dna table in the core db, so where analyses override slice->seq. Has simple headers with just the seq_region name. Also used by bwa in the RNA-seq analyses. Not masked
+    faidx_genome_file => catfile( $self->o('genome_dumps'), $self->o('species_name') . '_toplevel.fa' ),
+
+    'min_toplevel_slice_length' => 250,
+
+########################
+# Extra db settings
+########################
+
+    'num_tokens' => 10,
+
+########################
+# Executable paths
+########################
+    'star_path'    => '/homes/fergal/bin/STAR',
+    'scallop_path' => '/homes/fergal/bin/scallop',
+
```

Comment:
Could you please add `stringtie2_path`? It is unfortunately needed for the merge of the GTF

Reply:
Added.